### PR TITLE
required version of PHP is mentioned in installation guides

### PR DESCRIPTION
**Pull request by TomasLudvik**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Composer requires version of PHP 7.1 or newer when creating project
|New feature| No 
|[BC breaks](/docs/contributing/backward-compatibility-promise.md)| No 
|Fixes issues| ... 
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: docs/installation/installation-using-docker-linux.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Take a look at the article about [Monorepo](../introduction/monorepo.md) for mor
 
 ## Requirements
 * [GIT](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-* [PHP](http://php.net/manual/en/install.unix.php)
+* [PHP 7.1 - 7.3](http://php.net/manual/en/install.unix.php)
```
Comment (vitek-rostislav) [inline docs/installation/installation-using-docker-linux.md]: Hi, I would not mention the upper limit, we would need to change it every time we add support of new PHP :slightly_smiling_face:

Comment (TomasLudvik) [reply docs/installation/installation-using-docker-linux.md]: Hello @vitek-rostislav,
I got inspiration from native-installation.md where it is wrote same way :-) do you insist on updating it?

Comment (vitek-rostislav) [reply docs/installation/installation-using-docker-linux.md]: yes please :innocent: I would rather get inspired by the line below in the same doc where we mention minimum version for Docker, so I would put it this way:

```
* [PHP](http://php.net/manual/en/install.unix.php)
	* At least version **7.1 or higher**
```

Comment (TomasLudvik) [reply docs/installation/installation-using-docker-linux.md]: Hello @vitek-rostislav,

I have updated it as requested and also rebased on actual master.

**Pull request merged**